src/Interpreters/Context.cpp

```diff
@@ -1816,6 +1816,57 @@ std::optional<UInt16> Context::getTCPPortSecure() const
     return {};
 }
 
+// Small compile time FNV-1a 64 hashing.
+constexpr size_t compileTimeHashing(const std::string_view & s)
+{
+    size_t res = 0xcbf29ce484222325;
+    for (auto c : s)
+    {
+        res ^= c;
+        res *= 1099511628211;
+    }
+    return res;
+}
+
+constexpr size_t operator"" _h(const char * p, size_t s)
```

Comment:
From my perspective better to keep it simpler and just compare strings.

Reply:
Well... it's more concise (less code) and faster (compile time hash table). And IMO it's simple)